pombola/core/migrations/0011_organisation_alter_short_name.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
You could combine this into the previous migration so that the "create field" and "populate field" operations are together.